packages/venia-concept/src/components/MiniCart/cartOptions.js

```diff
@@ -116,48 +129,104 @@ class CartOptions extends Component {
             ? classes.modal_active
             : classes.modal;
 
+        const IncrementButtonFormApi = ({ formApi }) => {
+            return (
+                <button
+                    type="button"
+                    onClick={() => {
+                        this.incrementQty(formApi);
+                    }}
+                >
+                    +
+                </button>
+            );
+        };
+
+        const DecrementButtonFormApi = ({ formApi }) => {
+            return (
+                <button
+                    type="button"
+                    onClick={() => {
+                        this.decrementQty(formApi);
+                    }}
+                >
+                    -
```

Comment:
Please use the minus symbol "−" instead of hyphen "-".

Reply:
The buttons should have readable content (increase quantity, decrease quantity) or `aria-label` attributes, and SVG icons so they're not affected by typeface variance.

Reply:
Have replaced the "+" and "-" symbols with the corresponding svg's (Plus, Minus) imported from "react-feather".